### PR TITLE
Batch sdk integration

### PR DESCRIPTION
**Pull request by g2vinay**
Includes changes of 
Azure Batch base code migration.
Enabled Live Tests.

### CODE REVIEW

File: pom.client.build.xml [hunk 1]
```diff
@@ -35,6 +35,6 @@
 
   <modules>
     <module>./template/client</module>
-    <module>./keyvault/data-plane</module>
+    <module>./batch/data-plane</module>
```
Comment (JonathanGiles) [inline pom.client.build.xml]: This is probably not what you want to do :-)

Comment (g2vinay) [reply pom.client.build.xml]: It is fixed in pom.client.xml
This file is removed.

File: batch/data-plane/pom.xml [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+      <id>microsoft</id>
+      <name>Microsoft</name>
+    </developer>
+  </developers>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>azure-client-runtime</artifactId>
+      <version>1.6.4</version>
```
Comment (JonathanGiles) [inline batch/data-plane/pom.xml]: Should we reference the parent pom version here?

Comment (g2vinay) [reply batch/data-plane/pom.xml]: yeah moving them up.

File: batch/data-plane/src/main/java/com/microsoft/azure/batch/protocol/Jobs.java [hunk 3]
```diff
@@ -1016,7 +1016,7 @@
 
     /**
      * Lists the execution status of the Job Preparation and Job Release task for the specified job across the compute nodes where the job has run.
-     * This API returns the Job Preparation and Job Release task status on all compute nodes that have run the Job Preparation or Job Release task. This includes nodes which have since been removed from the pool. If this API is invoked on a job which has no Job Preparation or Job Release task, the Batch service returns HTTP status code 409 (Conflict) with an error code of JobPreparationTaskNotSpecified.
+     * This API returns the Job Preparation and Job Release task status on all compute nodes that have run the Job Preparation or Job Release task. This includes nodes which have since been removed from the pool. If this API is invoked on a job which has no Job Preparation or Job Release task, the Batch service returns HTTP status code 409 (Conflict) with an error code of JOB_PREPARATION_TASK_NOT_SPECIFIED.
```
Comment (xingwu1) [inline batch/data-plane/src/main/java/com/microsoft/azure/batch/protocol/Jobs.java]: The comment is generated by swagger spec. If you change this way, eventually, we have to manually update the comment every time we update swagger spec.

Comment (g2vinay) [reply batch/data-plane/src/main/java/com/microsoft/azure/batch/protocol/Jobs.java]: Yeah agreed, for the same reason, these changes were reverted. This specific version is outdated.
Latest version no longer has these changes: https://github.com/g2vinay/azure-sdk-for-java/blob/batch-sdk-integration/batch/data-plane/src/main/java/com/microsoft/azure/batch/protocol/Jobs.java

Comment (g2vinay) [reply batch/data-plane/src/main/java/com/microsoft/azure/batch/protocol/Jobs.java]: Latest changes can be seen here: https://github.com/Azure/azure-sdk-for-java/pull/2865/files

File: batch/data-plane/.travis.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+language: java
```
Comment (weshaggard) [inline batch/data-plane/.travis.yml]: Why do we need this travis.yml file?

Comment (JonathanGiles) [reply batch/data-plane/.travis.yml]: I noted this also. My thinking was that we would clean up docs, remove redundant files, etc after we get the repo imported.

Comment (weshaggard) [reply batch/data-plane/.travis.yml]: I'm fine with follow-up clean-up.

File: batch/data-plane/.gitignore [hunk 5]
```diff
@@ -0,0 +1,53 @@
+*.class
```
[the remark this answers is not in the document]
Comment (g2vinay) [reply batch/data-plane/.gitignore]: Will be removed in follow up cleanup.

File: batch/data-plane/.travis.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (g2vinay) [reply batch/data-plane/.travis.yml]: Will be removed in follow up cleanup.

File: batch/data-plane/LICENSE.txt [hunk 6]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
[the remark this answers is not in the document]
Comment (g2vinay) [reply batch/data-plane/LICENSE.txt]: Will be removed in follow up cleanup.

File: batch/data-plane/gulpfile.js [hunk 7]
```diff
@@ -0,0 +1,100 @@
+var gulp = require('gulp');
```
[the remark this answers is not in the document]
Comment (g2vinay) [reply batch/data-plane/gulpfile.js]: Will be removed in follow up cleanup.

**Pull request merged**